guilford_sent_component.yaml

```diff
@@ -0,0 +1,597 @@
+# select * from sent_component
+
+entityDefinitions:
+
+  offender:
+    fqn: "general.person"
+    entitySetName: "GuilfordOffenders"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "OFFENDER NC DOC ID NUMBER"
+      ol.datasource:
+        type: "ol.datasource"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "NC DPS"
+    name: "offender"
+
+  case:
+    fqn: "justice.case"
+    entitySetName: "GuilfordCourtCases"
+    propertyDefinitions:
+      j.CaseNumberText:
+        type: "j.CaseNumberText"
+        column: "COURT DOCKET NUMBER"
+      ol.type:
+        type: "ol.type"
+        column: "COURT TYPE CODE"
+    name: "case"
+
+  sentence:
+    fqn: "ol.sentence"
+    entitySetName: "GuilfordSentences_DPS"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["INMATE SENTENCE  STATUS CODE", "TYPE OF COUNT CODE", "ORIGINAL SENTENCE AUDIT CODE", "SENTENCE TYPE CODE_1", "SENTENCE TYPE CODE", "SENTENCE CONVICTION DATE"]
+      ol.type:
+        type: "ol.type"
+        column: "SENTENCE TYPE CODE"
+      ol.convictionresults:
+        type: "ol.convictionresults"
+        column: "COMPONENT DISPOSITION CODE"
+      ol.subtype:
+        type: "ol.subtype"
+        column: "SENTENCE TYPE CODE_1"
+      ol.status:
+        type: "ol.status"
+        column: "INMATE SENTENCE  STATUS CODE"
```

Comment:
are you sure there's two spaces in this column?

Reply:
yes